test/integration/pj-rehearse/master/core-services/cluster-pools/_config.yaml

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Autogenerated file?

Reply:
no, it has to exist so the pj-rehearse.sh test has something to work with